components/playground.tsx

```diff
@@ -236,17 +261,192 @@ export default function Playground({
 		<main className="grid lg:grid-cols-[2fr_3fr] gap-20 lg:gap-12 w-full max-w-4xl">
 			<Form {...form}>
 				<form
+					id="passportform"
 					onSubmit={form.handleSubmit(onSubmit)}
 					className="w-full flex flex-col gap-y-6"
 				>
+					<p>1. Register</p>
+
+					{userId && guildMember !== undefined ? (
+						<span>
+							<FormField
+								control={form.control}
+								name="sendToDb"
+								render={({ field }) => (
+									<FormItem className="space-y-3">
+										<FormLabel>Register this passport</FormLabel>
+										<FormDescription>
+											Register for an upcoming passport ceremony? SOME OTHER
+											TEXT HERE (NTS: Ask Matthew Later -Michael)
```

Comment:
Let's do:

"Register for an hour-long passport ceremony to turn your data page into a real-life passport."

Reply:
Resolved in [f122838](https://github.com/purduehackers/passport-issuing-office/pull/52/commits/f122838cfd8f85d9ac4a4e424a901a1a1e111c1e)